services-js/commissions-app/src/client/common/Checkboxes.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { PUBLIC_CSS_URL } from '@cityofboston/react-fleet';
+import fetchCommissions, {
+  Commission,
+} from '../../client/graphql/fetch-commissions';
+import Head from 'next/head';
+
+export interface Props {
+  commissions: Commission[];
+}
+
+export default class IndexPage extends React.Component {
+  static async getInitialProps(): Promise<Props> {
```

Comment:
Getting the commissions should be the job of the `ApplyPage`. This component just needs to focus on rendering one checkbox.